### DISCUSSION TITLE
Multiple settings files in .config/ for PCManFM-Qt

### DISCUSSION THREAD

**Discussion by isf63** [lxqt/lxqt#2500] [Q&A] [answered]
There are multiple settings files for PCManFM-Qt under my setup's ~/.config . I checked whether this was the case for other LXQt components, but it doesn't appear so.

There is also /usr/share/pcmanfm-qt/lxqt/settings.conf - I understand these are the default settings in the absence of ones defined in ~/.config

The files under ~/.config are are:
```
~/.config/pcmanfm-qt/
├── default
│   ├── recent-files.conf
│   └── settings.conf
└── lxqt
    ├── desktop-items-0.conf
    ├── recent-files.conf
    └── settings.conf
```

**Comment by tsujan** [comment]
So, what's the question?!

**Reply by isf63** [reply]
Whoops, I omitted that, ha.
I am wondering which of these under ~/.config control user settings. And why there are two instances under ~/.config (given that a "default" config should be provided by /usr/share/pcmanfm-qt/ .

**Comment by tsujan** [comment, marked as answer]
> ... which of these under ~/.config control user settings....

All of them; they cover different functionalities of the app in a clean way.

To know what each one is for, you could just open it (but don't edit it). Their names are also explanatory.

> And why there are two instances

One is used under an LXQt session, the other one outside LXQt. PCManFM-Qt doesn't depend on other LXQt components but is a separate app tuned to LXQt.

EDIT:
Also see `pcmanfm-qt --help` → `-p, --profile <PROFILE>`
